### DISCUSSION TITLE
Profile not get reflacted with commits that where done in PR

### DISCUSSION THREAD

**Discussion by PRATHAM-SPS** [community/community#69671] [Other Feature Feedback, Questions, & Ideas]
Bug

Hello Everyone I am contributing in openvino since last 3 months with one PR which contains 230 commits , today it get merged in there repo but that 230 commits are still not reflacted on my account. What is the solution for it or I have to wait?

**Comment by vishalkrishna8** [comment]
Hello @PRATHAM-SPS , 
You can check the commit history of the repository to confirm that your commits are indeed part of the repository's history. This can be done by visiting the repository and looking at the commit history or it can sometimes take up to 24 hours for contributions to appear on your profile. You can wait for a bit and check again later.

**Reply by PRATHAM-SPS** [reply]
Hello @vishalkrishna8 commits are present in the history , lets wait for 24 hrs.

**Reply by vishalkrishna8** [reply]
Also ensure that the repository where you made the contributions is a public repository. Contributions to private repositories do not appear on your public GitHub profile.

**Reply by PRATHAM-SPS** [reply]
yeah, its an open source project

**Reply by vishalkrishna8** [reply]
If you found this useful than you can mark my reply as an answer

**Comment by santoshvandari** [comment]
It appears that you've contributed to an open-source project on GitHub and your profile is not reflecting the 230 commits made in a recently merged pull request (PR). Here's some information and steps to help address this issue:

- GitHub Contribution Graph: GitHub's contribution graph, which shows your commit history on your profile, is updated periodically. Commits and contributions may not appear on your profile immediately after a PR is merged. Sometimes, it can take a few hours to a day for the graph to update.

- Wait for Updates: In most cases, you should wait for some time (usually a few hours to a day) for the contribution graph to update and reflect the merged commits. It's a common occurrence for these updates to take a bit of time.

- Check Privacy Settings: Ensure that your GitHub email address is associated with the commits made in the PR. GitHub associates commits with your profile based on the email address used in the commits. Also, make sure your contributions are not marked as "private" in your profile settings.

- Contact GitHub Support: If, after a reasonable amount of time, you still don't see the contributions reflected on your profile, you can consider reaching out to GitHub Support for assistance. They can investigate and help resolve any potential issues with your profile updates. To contact GitHub Support, you can visit [GitHub Support](https://support.github.com/contact) and describe the issue you're facing. They will be able to provide more specific guidance and assistance.

In most cases, waiting for a bit of while should resolve the issue as GitHub updates your profile to reflect your contributions accurately. However, if the problem persists, reaching out to GitHub Support is a good next step.

I hope this information helps, and your contributions will be properly reflected on your profile soon.

**Reply by PRATHAM-SPS** [reply]
Still it is not reflected  its been a more than  24 hrs🙂

**Reply by vishalkrishna8** [reply]
May be your email address associated with your GitHub account doesn't match the email address used for the commits in the repository. GitHub associates commits with your account based on the email address.

**Reply by PRATHAM-SPS** [reply]
Both mails are same

**Comment by PRATHAM-SPS** [comment]
Hello @santoshvandari @vishalkrishna8 do you have any solution on it , its still not reflacted on my profile.

**Reply by vishalkrishna8** [reply]
You must have made some mistake, check your contributions graph, you can click on the contributions graph on your profile page to see if it shows the correct number of contributions.

**Reply by santoshvandari** [reply]
Dear Sir, I have checked your profile and found that it is showing in your PR request in the contribution graph but not your commitment. I suggest you check the email configuration of git using the command `git config --global user.email` (Note: use this command in that machine from where you made the commitment). If the resulting email matched with the GitHub account email. Then you can report it as a bug to GitHub otherwise it was your mistake for misconfiguration of the account. 

Please Read it out: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

**Comment by PRATHAM-SPS** [comment]
> Dear Sir, I have checked your profile and found that it is showing in your PR request in the contribution graph but not your commitment. I suggest you check the email configuration of git using the command `git config --global user.email` (Note: use this command in that machine from where you made the commitment). If the resulting email matched with the GitHub account email. Then you can report it as a bug to GitHub otherwise it was your mistake for misconfiguration of the account.
> 
> Please Read it out: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

Hey I have the same mail ID, and I have made many other contribution from same machine it shows up in the profile but this PR is causing problem , @santoshvandari  please can u explain me how should I report this bug.

**Reply by PRATHAM-SPS** [reply]
@santoshvandari @davidstaheli @mgriffin @zackfern @martinwoodward 
**PR Link :-** https://github.com/openvinotoolkit/openvino/pull/18512 

**Total 230 commits in this PR itself , but on my profile its showing only 50.**
**I have verified my email id it is same as associated with git account.**

**Reply by PRATHAM-SPS** [reply]
Hello @santoshvandari  just a friendly reminder.

**Reply by santoshvandari** [reply]
I don't have any more idea regarding this. Sorry
